impeller/entity/shaders/geometry/points.comp

```diff
@@ -0,0 +1,67 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <impeller/texture.glsl>
+#include <impeller/types.glsl>
+
+// I dunno.
```

Comment:
🤷‍♂️

Reply:
The way we use these shaders, these values are set by the `GridSize` API and get dropped.

I've tried to specify something meaningful if we were to use them, but you could just say `See PointFieldGeometry::GetPositionBuffer` or something.

Reply:
Done